app/Controllers/configureController.php

```diff
@@ -131,6 +131,7 @@ public function readingAction(): void {
 			FreshRSS_Context::userConf()->lazyload = Minz_Request::paramBoolean('lazyload');
 			FreshRSS_Context::userConf()->sides_close_article = Minz_Request::paramBoolean('sides_close_article');
 			FreshRSS_Context::userConf()->sticky_post = Minz_Request::paramBoolean('sticky_post');
+			FreshRSS_Context::userConf()->mark_read_button = Minz_Request::paramStringNull('mark_read_button') ?? 'big';
```

Comment:
I do not understand the issue here:

```
 Check failure on line 134 in app/Controllers/configureController.php
GitHub Actions / tests

Property FreshRSS_UserConfiguration::$mark_read_button ('big'|'none'|'small') does not accept string.
```

Reply:
It is because you have not checked that the string is only 'big'|'none'|'small'.
You may do so with e.g. `in_array()`

Reply:
Thanks for the help.
Would it make sense to have a function similar to `Minz_Request::paramStringNull` that have the check implemented too?

Reply:
> Would it make sense to have a function similar to `Minz_Request::paramStringNull` that have the check implemented too?

I did a quick test but did not found how to do that in a way that is understood by PHPStan, so keeping it simple for now, but help welcome with another proposition that works.